commit 905c2678a3a2ba6afee12ab31446db3054ccc741
Author: Joachim Wiberg <troglobit@gmail.com>
Date:   Mon Mar 8 09:27:08 2021 +0100

    Fix path to fallback console and handle leading /dev from cmdline
    
    Signed-off-by: Joachim Wiberg <troglobit@gmail.com>

diff --git a/src/helpers.c b/src/helpers.c
index 3486879..8540825 100644
--- a/src/helpers.c
+++ b/src/helpers.c
@@ -103,6 +103,9 @@ static void add_console(char *cons)
 	if (ptr)
 		*ptr = 0;
 
+	if (!strncmp(cons, "/dev/", 5))
+		cons += 5;
+
 	snprintf(path, sizeof(path), "/dev/%s", cons);
 
 	/* check for and skip duplicates */
@@ -131,7 +134,7 @@ static void find_active_consoles(void)
 	if (!fp) {
 	fallback:
 		_d("Using fallback /dev/console");
-		add_console("/dev/console");
+		add_console("console");
 		return;
 	}
 
